stuff/cron/assign_badges.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/python3
+
+''' Assigns users badges and creates the notifications.'''
+
+import argparse
+import json
+import logging
+import os
+
+import MySQLdb
+
+import lib.db
+import lib.logs
+
+
+BADGES_PATH = os.path.abspath(os.path.join(__file__, '..', '..',
+                                           '..', 'frontend/badges'))
+
+
+def delete_read_notifications(cur: MySQLdb.cursors.DictCursor):
+    '''Deletes all read notifications'''
+    cur.execute("""
+        DELETE FROM
+            `Notifications`
+        WHERE
+            `read` != 0;""")
+    logging.info('Deleted all already read notifications, %s in total.',
+                 cur.rowcount)
+
+
+def get_all_owners(badge: str, cur: MySQLdb.cursors.DictCursor):
```

Comment:
puedes agregarle tipos a los valores de retorno?

```suggestion
def get_all_owners(badge: str, cur: MySQLdb.cursors.DictCursor) -> Set[int]:
```

y arriba

```python
from typing import Set
```

igual al resto de las funciones.